example/README.md

```diff
@@ -0,0 +1,60 @@
+# Example Extension
+
+## Creating a New Layout
+
+If you desire to use your favorite keyboard layout but it is not available by
+default in Chrome OS or in this repository then a new extension will need to be
+created. In this directory you can find some examples to use as a template for
+creating your own. Please feel free to submit a pull request to share your
+extension with the world.
+
+### Preparation
+
+The first thing to do is determine if you can take the simple route or if your
+extension will require more effort. If your layout is already in the version of
+[xkeyboard-config](https://github.com/freedesktop/xkeyboard-config) that is
+available in Chrome OS you are nearly ready.
+
+On your Chrome OS device, open a new browser tab and go to `chrome://version`.
+Find both the "Google Chrome" and the "Platform" version numbers. My device, as
+I write this, is "Google Chrome" **85**.0.4183.84 and "Platform" **13310**.59.0.
+
+Navigate to the [Chromium source
```

Comment:
It might be simpler to link to https://gitlab.freedesktop.org/xkeyboard-config/xkeyboard-config/-/blob/xkeyboard-config-2.27/rules/base.xml citing  https://source.chromium.org/chromiumos/chromiumos/codesearch/+/main:src/third_party/chromiumos-overlay/x11-misc/xkeyboard-config/ for the version. The ebuild changes infrequently, so the link can be updated as needed.




https://gitlab.freedesktop.org/xkeyboard-config/xkeyboard-config/-/blob/xkeyboard-config-2.27/rules/base.xml